tests/test_fields.py

```diff
@@ -171,6 +187,14 @@ def test_download(self):
             self.assertEqual(serializer.data['image'], encoded_source)
         finally:
             os.remove('im.jpg')
+        
+    def test_not_found(self):
+        image = DownloadableBase64Image('file_does_not_exist.jpg')
+        serializer = DownloadableBase64ImageSerializer(image)
+        callable = lambda: serializer.data
+        
+        self.assertRaises(FileNotFoundError, callable)
+        
```

Comment:
What about mocking `django.core.files.base.File.open` here and using `side_effect` we can make it raise a `FileNotFoundError`.